src/bun.js/bindings/sqlite/JSSQLStatement.cpp

```diff
@@ -742,15 +988,16 @@ static JSC::JSValue rebindStatement(JSC::JSGlobalObject* lexicalGlobalObject, JS
         return jsNumber(0);
     }
 
-    if (count != max) {
-        throwException(lexicalGlobalObject, scope, createError(lexicalGlobalObject, "Expected " + String::number(max) + " values, got " + String::number(count)));
+    int required = bindings.count;
+    if (count != required) {
+        throwException(lexicalGlobalObject, scope, createError(lexicalGlobalObject, makeString("SQLite query expected "_s, required, " values, received "_s, count)));
```

Comment:
after throw should we return empty/zero instead of undefined?

Reply:
Both work. We aren't consistent right now